app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -144,6 +198,44 @@ def gather_data_for_this_region(arm_service, method = "list")
         results.flatten
       end
 
+      def parse_security_group(security_group)
+        uid = security_group.id
+
+        description = [
+          security_group.name,
+          security_group.resource_group,
+          security_group.location
+        ].join(' - ')
+
+        new_result = {
+          :type        => self.class.security_group_type,
+          :ems_ref     => uid,
+          :name        => security_group.name,
+          :description => description
+        }
+
+        return uid, new_result
+      end
+
+      def parse_firewall_rules(sg)
+        rule_array = []
+
+        sg.properties.security_rules.each do |rule|
+          new_result = {
+            :name            => rule.name,
+            :host_protocol   => rule.properties.protocol.upcase,
+            :port            => rule.properties.destination_port_range.split('-').first,
+            :end_port        => rule.properties.destination_port_range.split('-').last,
+            :direction       => rule.properties.direction,
+            :source_ip_range => rule.properties.source_port_range,
+          }
```

Comment:
Ah, that should probably be sourceAddressPrefix, which the docs describe as "CIDR or source IP range or * to match any IP."